src/scripts/install.py

```diff
@@ -63,10 +63,23 @@ def log_level():
 
     return (options, args)
 
+def prepend_destdir(path):
+    """
+    Needed because os.path.join() discards the first path if the
+    second one is absolute, which is usually the case here. Still, we
+    want relative paths to work and leverage the os awareness of
+    os.path.join().
+    """
+    try:
+        destdir = os.environ['DESTDIR']
+    except KeyError as e:
+        destdir = ""
+    return os.path.join(destdir, path.lstrip('/'))
+
 def makedirs(dirname, exist_ok = True):
     try:
         logging.debug('Creating directory %s' % (dirname))
-        os.makedirs(dirname)
+        os.makedirs(prepend_destdir(dirname))
```

Comment:
Since this is a generic filesystem helper method, I think it is cleaner to prepend destdir in the caller than here. This might lead to undesired behavior if for create other directories.

Additionally we get proper log messages including the full path with destdir